### ISSUE TITLE
import/named doesnt work with flow

### ISSUE THREAD

**Issue by bradennapier** [benmosher/eslint-plugin-import#1133]
```javascript

import type { PluginIDType } from '../plugin';
```
gets import/named error

using flow... the file has

```javascript
export opaque type PluginIDType: string = string;
```

(it also doesnt work with any regular types either FYI its not just opaque type - this eslint plugin just fails pretty often with flow used)

Flow works flawlessly and knows the type but this errors out

**Comment by syymza** [comment]
I believe this is fixed by #1106 . @ljharb can you please release, the fix has been there for more than 3 weeks but is not yet out....

**Comment by ljharb** [comment]
Thanks, then this can be closed.

A release will be made when there's time to make one.
